tools/warming_levels.ipynb

```diff
@@ -170,14 +178,24 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "98fac248-0826-4ad5-9db3-9094bb750390",
+   "id": "d5fe6825-98a7-4d72-a629-0abae3f7eb4c",
    "metadata": {},
    "outputs": [],
    "source": [
     "%%time\n",
     "wl.calculate()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "06d49aa0-8a4a-4052-909d-004e6d26b159",
+   "metadata": {},
+   "source": [
+    "And let's set a few other things for this example, in case we forget to do so above.\n",
```

Comment:
I think this change may be showing because a cell moved or something, this was already written in the notebook before.